### PR TITLE
Updated gitignore to allow migrations folder

### LINKED ISSUE #88: Update gitignore to allow migrations

### PR DESCRIPTION
**Pull request by Monal5031**
Fixes #88

### CODE REVIEW

**Comment by yashLadha** [comment]
Update the PR title

**Comment by Monal5031** [comment]
Done @yashLadha sir

**Comment by yashLadha** [comment]
We should allow migartions folder complete. Follow the [thread](https://stackoverflow.com/questions/33784362/django-migrations-best-practice). Can you make change according to it.

File: .gitignore [hunk 1]
```diff
@@ -8,3 +9,6 @@
 wsgi/media/photologue/photos/
 **/migrations/
```
Comment (yashLadha) [inline .gitignore]: Allow the migrations.

**Comment by Monal5031** [comment]
@yashLadha I can do it no problem, but it may cause merge conflicts sometimes.

**Comment by yashLadha** [comment]
Yaa, that is an issue, but it is considered a good practice to include migration folder. We have to look over it. Views @Prakash2403 @singhpratyush

**Comment by Monal5031** [comment]
@yashLadha We can allow it, if not everyone uses `makemigrations` locally as they will be uploaded once only.

**Comment by yashLadha** [comment]
That has to be included in `README`. And also if some make some changes to model them also the necessary migration files need to be uploaded, so that it can be easily used by other contributors.

**Comment by yashLadha** [comment]
Them there is a change in `README` required.

**Comment by Prakash2403** [comment]
@yashLadha migration file will be generated when user will run `python manage.py makemigrations`. So, I think there is no need of including them.

**Comment by Monal5031** [comment]
@yashLadha @Prakash2403 So should I allow only `__init__.py` or all.
Allowing only `__init__.py` has an advantage though, that no merge conflicts and everyone can makemigrations according to them and not affect others. your views?

**Comment by yashLadha** [comment]
What the migration files or the instruction. I have read that it is written in the official documentation as a good practice to include migration files. So that other contributors can easily work on latest changes.

**Comment by Prakash2403** [comment]
@yashLadha @Monal5031 Let's include all migration files.

**Comment by yashLadha** [comment]
Update the Instruction in `README` accordingly.

**Comment by Prakash2403** [comment]
Although some problems may be there (https://docs.djangoproject.com/en/1.11/topics/migrations/#version-control)[Read this]

**Comment by Monal5031** [comment]
@yashLadha @Prakash2403 Updated. Please review.

File: README.md [hunk 2]
```diff
@@ -43,3 +43,7 @@ $ python manage.py makemigrations
 $ python manage.py migrate
 $ python manage.py runserver
 ```
+
+* Note:
```
Comment (yashLadha) [inline README.md]: Make them Bold

Comment (Monal5031) [inline README.md]: Should I also fix the line length errors in migration files?

Comment (yashLadha) [inline README.md]: May be we should not make changes in them, because they are autogenerated files. It might break.

Comment (Monal5031) [inline README.md]: Okay, 👍

**Comment by yashLadha** [comment]
Does it require `makemigrations` instruction, when migration files are supplied? May be not? @Monal5031 make channges accordingly.

**Comment by Monal5031** [comment]
@yashLadha done sir.

**Comment by yashLadha** [comment]
Let @singhpratyush  Sir, also review the PR.

**Pull request merged**